iceoryx_dust/test/moduletests/test_cxx_string_conversion.cpp

```diff
@@ -108,4 +110,618 @@ TYPED_TEST(StdString_test, StringOfSizeCapaToSTDStringConvResultsInSizeCapa)
     EXPECT_THAT(testString2.size(), Eq(STRINGCAP));
     EXPECT_THAT(testString2.c_str(), StrEq(testString1.substr(0, STRINGCAP)));
 }
+
+TYPED_TEST(StdString_test, UnsafeAppendEmptyStdStringWorks)
```

Comment:
@elBoberido @FerdinandSpitzschnueffler These tests are not new, just the `std::string` part separated from the other test cases.